vllm/v1/core/scheduler.py

```diff
@@ -557,6 +559,7 @@ def update_from_output(
                 # Otherwise, we ignore the sampler output for the request.
                 request.num_computed_tokens += num_tokens_scheduled
                 assert request.num_computed_tokens <= request.num_tokens
+                spec_decoding_stats.num_emitted_tokens += num_tokens_scheduled
```

Comment:
I'm a bit uncertain whether to include this in the count ... it matches what I was getting at the rejection sampler level, but ... I guess it depends what we want the system efficiency metric to mean.

Will dig into it, but thoughts welcome!